### PR TITLE
New Version: Astronomer.Astro version 1.17.0

### PR DESCRIPTION
**Pull request by neel-astro**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111506&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Astronomer.Astro
+PackageVersion: 1.19.0
```
Comment (CoolPlayLin) [inline manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.yaml:5]:
```suggestion
PackageVersion: 1.17.0
```

File: manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Astronomer.Astro
+PackageVersion: 1.17.0
+InstallerLocale: en-US
+InstallerType: portable
```
Comment (CoolPlayLin) [inline manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.installer.yaml:7]:
```suggestion
InstallerType: nullsoft
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

File: manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.installer.yaml (hunk 2 above)
Comment (neel-astro) [reply manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.installer.yaml:7]: Why would we need a `nullsoft` installer type? This is a standalone exe file without any installer.

File: manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.yaml (hunk 1 above)
Comment (neel-astro) [reply manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.yaml:5]: Thanks for catching it, typo on my end

**Comment by CoolPlayLin** [comment]
image
I think you should pack it as a zip, Because Winget does not support such a InstallerType

**Comment by neel-astro** [comment]
> I think you should pack it as a zip, Because Winget does not support such a InstallerType

@CoolPlayLin I think Winget does support portable installer type based on the documentation: https://learn.microsoft.com/en-us/windows/package-manager/winget/#supported-installer-formats, and also many other packages use portable installer type as of now in the repo.

Good to know that zip is now supported, I can update the release on the Astro side to pack it in the zip for future releases, but I won't be able to do it for 1.17.0 since that has already been released.

**Comment by CoolPlayLin** [comment]
I tested just not, It can be installed.

**Comment by neel-astro** [comment]
> I tested just now, It can be installed.

As in it can be installed just fine with installer-type portable I assume? It's been working fine with the portable installer type all this while, so I think it should work fine with this new release as well.

File: manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.installer.yaml (hunk 2 above)
Comment (stephengillie) [reply manifests/a/Astronomer/Astro/1.17.0/Astronomer.Astro.installer.yaml:7]: This appears to have been updated.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**